assets/javascripts/swift-evolution.js

```diff
@@ -9,6 +9,11 @@
 // ===---------------------------------------------------------------------===//
 'use strict'
 
+let EVOLUTION_METADATA_URL = 'https://download.swift.org/swift-evolution/proposals.json'
+let GITHUB_BASE_URL = 'https://github.com/'
+let REPO_PROPOSALS_BASE_URL = GITHUB_BASE_URL + 'apple/swift-evolution/blob/main/proposals'
+let UFF_INFO_URL = '/blog/using-upcoming-feature-flags/'
```

Comment:
Did you mean `const` here?

```suggestion
const EVOLUTION_METADATA_URL = 'https://download.swift.org/swift-evolution/proposals.json'
const GITHUB_BASE_URL = 'https://github.com/'
const REPO_PROPOSALS_BASE_URL = GITHUB_BASE_URL + 'apple/swift-evolution/blob/main/proposals'
const UFF_INFO_URL = '/blog/using-upcoming-feature-flags/'
```

Reply:
Yes, thanks. I had to redo my original change locally and forgot to make everything const.

Thank you!